test/typescript/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  rules: {
+    'no-unused-expressions': 'off',
+    '@typescript-eslint/no-namespace': 'off',
+    '@typescript-eslint/no-unused-vars': 'off'
+  }
+}
```

Comment:
These rules aren't useful for the TypeScript tests because the code doesn't actually run, it's just meant to be typechecked